apps/api/src/app/auth/google.strategy.ts

```diff
@@ -11,6 +11,7 @@ import { AuthService } from './auth.service';
 export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
   public constructor(
     private readonly authService: AuthService,
+    // @ts-expect-error: Property is being used in the constructor
     private readonly configurationService: ConfigurationService
```

Comment:
It could work without `@ts-expect-error` when you avoid declaring it as a class property:
```
configurationService: ConfigurationService
```

Reply:
Something like this also worked :)
```
public constructor() {
    const configurationService = inject(ConfigurationService);
    super(...)
}
```